### PR TITLE
feat(ckanext-util): adding more azure secrets

### PR DESCRIPTION
**Pull request by Locutus66**
- Adding more azure secrets that relate to ckanext-util
- Increased chart version to 0.0.8

### CODE REVIEW

**Comment by sylus** [comment]
@blairdrummond look good to you?

File: stable/ckan/templates/deploy/ckan.yaml [hunk 1]
```diff
@@ -259,6 +259,21 @@ spec:
                 secretKeyRef:
                   name: {{ include "ckan.fullname" . }}-util
                   key: azureClientSecretUtil
+            - name: AZURE_CLIENT_ID
+                valueFrom:
```
Comment (blairdrummond) [inline stable/ckan/templates/deploy/ckan.yaml:263]: the `valueFrom` shouldn't be indented (ditto for next two items)

**Review by blairdrummond — CHANGES_REQUESTED** [review]
Looks ok but minor formatting errors

File: stable/ckan/templates/deploy/ckan.yaml (hunk 1 above)
Comment (Locutus66) [reply stable/ckan/templates/deploy/ckan.yaml]: Good catch - fixed.

**Review by blairdrummond — APPROVED** [review]
LGTM! CC @sylus

**Pull request merged**